source/setup/alternatives.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Alternative persona setups
+layout: layout
+---
+
+# Alternative persona setups
+
+Our personas have inspired other people to create their own version.
+Here are some of the those other versions.
+
+
+## [HMRC's Virtual Empathy Hub](https://personas-prototype.herokuapp.com/)
+
+[HMRC](https://www.gov.uk/government/organisations/hm-revenue-customs)'s Digital Inclusion and Accessibility Standards team created a version of these personas that move the assistive technologies and simulations into the website itself.
+That means you cannot install these anywhere and use them on other websites, but it makes it easier to share the general idea and go through the various exercises without needing any other tools.
```

Comment:
```suggestion

That means you cannot install these anywhere and use them on other websites, but it makes it easier to share the general idea and go through the various exercises without needing any other tools.
```

Is there supposed to be an additional line here?

Reply:
I don't think so. I will push a change that includes removing that newline so it's all in one line.